components/layouts/content-slider/content-slider.hbs

```diff
@@ -1,193 +1,169 @@
+          class="content-slider__controls-left"
+        ></a>
+        <a href="" aria-label="next" class="content-slider__controls-right"></a>
       </div>
-      <!-- .content-slider__block -->
-
-      <div class="content-slider__block">
-
-        <a href="#" class="content-block--clickable">
-          <div class="content-block__upper">
-            <img
-              class="content-block__image"
-              src="https://via.placeholder.com/535x300"
-              role="presentation"
-              alt=""
-            />
-            <time class="content-block__date" datetime="2021-06-10">
-              <span
-                class="content-block__date-month"
-                aria-label="June"
-              >Jun</span>
-              <span class="content-block__date-day">10</span>
-            </time>
-          </div><!-- .content-block__upper -->
-
-          <div class="content-block__lower">
-            <p class="content-block__subtitle">Digitalks Webinar</p>
-            <h4 class="content-block__title">RSVP: This is a featured event that
-              is topical and relevant.</h4>
-
-            <p class="content-block__text">Lorem ipsum dolor, sit amet
-              consectetur adipisicing elit. Voluptate sit id harum aliquid ea.
-            </p>
-            <div class="button--medium-primary content-block__button">Learn More</div>
-          </div><!-- .content-block__lower -->
-        </a><!-- .content-block--clickable -->
 
+      <div class="content-slider__cards-list">
+        <div class="content-slider__block">
+          <a href="#" class="content-block--clickable">
+            <div class="content-block__upper">
+              <img
+                class="content-block__image"
+                src="https://via.placeholder.com/535x300"
+                role="presentation"
+                alt=""
+              />
+
+              <time class="content-block__date" datetime="2021-06-10">
+                <span
+                  class="content-block__date-month"
+                  aria-label="June"
+                >Jun</span>
+                <span class="content-block__date-day">10</span>
+              </time>
+
+            </div><!-- .content-block__upper -->
+
+            <div class="content-block__lower">
+              <p class="content-block__subtitle">Digitalks Webinar</p>
+              <h4 class="content-block__title">RSVP: This is a featured event that
+                is topical and relevant.</h4>
+              <p class="content-block__text">Lorem ipsum dolor, sit amet
+                consectetur adipisicing elit. Voluptate sit id harum aliquid ea.
+              </p>
+              <div class="button--medium-primary content-block__button">Learn More</div>
+            </div><!-- .content-block__lower -->
+            
+          </a><!-- .content-block--clickable -->
+        </div>
+        <!-- .content-slider__block -->
+        <div class="content-slider__block">
```

Comment:
Not a blocker on this, but might be a way to make this code a little simpler to abstract this repeated chunk of code into a file you just include three times.